run_rustc/Makefile

```diff
@@ -109,19 +119,21 @@ all: $(BINDIR)rustc
 all: $(BINDIR)cargo
 
 # Helper rules to ensure that mrustc has built rustc and cargo
-../output$(OUTDIR_SUF)/rustc:
-	$(MAKE) -C ../ output$(OUTDIR_SUF)/rustc -j 3
-../output$(OUTDIR_SUF)/cargo:
-	$(MAKE) -C ../ output$(OUTDIR_SUF)/cargo -j 3
+$(MINICARGO):
+	$(MAKE) -C ../ bin/minicargo$(EXESUF)
+$(MRUSTC_RUSTC):
+	$(MAKE) -C ../ output$(OUTDIR_SUF)/rustc
+$(MRUSTC_CARGO):
+	$(MAKE) -C ../ output$(OUTDIR_SUF)/cargo
```

Comment:
Probably worth keeping these using the default paths - So it errors out properly if you override `MRUSTC_RUSTC` to something non-exoistant